frontend/src/App.js

```diff
@@ -74,11 +74,10 @@ const App = () => (
                     height: 100%;
                   }
                   @font-face {
-                    font-family: 'robotoregular';
-                    src: url('fonts/roboto-regular-webfont.woff2')
-                        format('woff2'),
-                      url('fonts/roboto-regular-webfont.woff') format('woff');
-                    font-weight: normal;
```

Comment:
Are we still using `fonts/roboto-regular-webfont.woff2` and `'fonts/roboto-regular-webfont.woff'` or can we delete these files?